src/dv-object/domain/models/DvObjectOwnerNode.ts

```diff
@@ -0,0 +1,12 @@
+export interface DvObjectOwnerNode {
```

Comment:
I don't think dv-object should be considered as a sub-domain in the package. I would place this file and any other dvObject related file in the core subdomain, as the dvObject is a concept shared across the different subdomains (collections, files and datasets).